Tests/Realm.Tests/Sync/SyncTestHelpers.cs

```diff
@@ -92,6 +92,8 @@ public static void RunBaasTestAsync(Func<Task> testFunc, int timeout = 30000)
 
         public static string GetVerifiedUsername() => $"realm_tests_do_autoverify-{Guid.NewGuid()}";
 
+        public static string GetUnconfirmedUsername() => $"realm_tests_do_not_confirm-{Guid.NewGuid()}@g.it";
```

Comment:
`RetryCustomConfirmation` will fail if the input doesn't look like a proper email. We could add the same to `GetVerifiedUsername` for consistency